package.json

```diff
@@ -264,7 +264,7 @@
     "metals-languageclient": "0.1.14",
     "promisify-child-process": "^3.1.3",
     "semver": "^7.1.2",
-    "vscode-languageserver-protocol": "3.15.2"
+    "vscode-languageserver-protocol": "3.15.0-next.6"
```

Comment:
I believe this was my mistake, and I updated it again even though we bumped it back before. If this is necessary to stay on this version, can we add a short note saying that so I don't accidentally bump this again if dependebot comes knocking.

Reply:
This is quite messy, since different dependencies depend on slightly different versions.
This version at least doesn’t surface any error in coc-Metals (while it still does in the dependencies, but we can ignore it for now)

Reply:
I’ve chosen to use the same version as coc.nvim so maybe we can just remove the dependency and rely on the transitive one. Not sure what’s the best option honestly... I’ll give it some more thoughts

Reply:
Sounds good.

Reply:
Ok, for this specific dependency the divergence was caused by metals-languageclient, so I've published a new version of it that uses exactly this version (which is the same coc.nvim uses) so that they're all happy.

```
❯ yarn list --pattern vscode-languageserver-protocol
yarn list v1.17.3
└─ vscode-languageserver-protocol@3.15.0-next.6
```

Reply:
Sounds good 👍